index.bs

```diff
@@ -335,9 +342,23 @@ Note: [Feature Policy](https://docs.google.com/document/d/1k0Ua-ZWlM_PsFCFdLMa8k
 should allow securely relaxing those restrictions once it matures.
 
 
+<h4 id="loosing-focus">Loosing Focus to a Different Origin</h4>
+
+When a different [=top-level browsing context=] of a different [=origin=] [=gains focus=],
+or when [=nested browsing context=] of a different [=origin=] [=gains focus=],
+for example when the user carries out an in-game purchase
+using a third party payment service from within an iframe,
+they are at risk of skimming attacks from [=top-level browsing contexts=] of a different [=origin=]
+that are operating sensors.
```

Comment:
Is 'operating sensors' correct term? Maybe can be synced somehow with |active sensors set| for top level browsing context?